### ISSUE TITLE
Different font settings for mobile vs desktop

### ISSUE THREAD

**Issue by sharik14** [kepano/obsidian-minimal#505]
The Minimal theme settings pane has a setting “Small font size”, default 13. I’d like to leave it at 13 for the desktop, but on my iOS devices have it set to 20. I'm not sure if there's a way to do this. At present it seems to ping-pong – when I open an iOS device and configure it to 20, but leave it unchanged on the desktop, everything works for a while, but sometimes (I think when I use the desktop app) it keeps resetting it back.

The only other way I can think of to get this behavior is to use an entirely different .obsidian folder on mobile, e.g. .obsidian-mobile -- which I really don't want to do, as I like sharing most of my settings across platforms. (I think it *might* also be possible with Obsidian selective settings Sync, although not positive; but in any case I'm using a shared folder [iCloud Drive] to sync my files between devices.)

I found the setting in question. It seems to be under .obsidian/plugins/obsidian-minimal-settings/data.json. I see a key textSmall. But this file appears to be shared btw desktop and mobile.

Would it be possible to add different settings for textSmall on mobile vs desktop, somehow?  This font is really used to control the font of system widgets, which mean something totally different on mobile. So it seems reasonable to have a different setting.

Thank you!

**Comment by toph-allen** [comment]
Yeah, this is really necessary for me to keep using this theme — managing separate settings for mobile and desktop is really untenable.

**Comment by DRhoadey** [comment]
Not sure if this is essentially the same feature request, but I'd love a way to set line widths for desktop and mobile separately. On mobile I'd prefer to have my "normal width" text run the entire width of the screen, but I like a narrow normal line width on desktop.

**Comment by 702573N** [comment]
Obviously, there are several motivations to have different theme settings for mobile and desktop operating systems. My need here is rather to define different themes or different background contrasts for desktop, or mobile. On mobile I prefer true black, while on desktop I prefer the default setting.

**Comment by kepano** [comment]
This can be achieved using selective sync in Obsidian Sync

Since Minimal has hundreds of settings duplicating all of them for mobile would add an enormous amount of code and complexity
